content/en/containers/guide/operator-eks-addon.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Installing Agent on AWS EKS using Operator Add-on
+kind: guide
+aliases:
+ - /agent/guide/operator-eks-addon
+further_reading:
+  - link: 'agent/kubernetes/log'
+    tag: 'Documentation'
+    text: 'Datadog and Kubernetes'
+---
+
+Datadog Operator can be installed on EKS cluster as an add-on. Add-ons provide installation and management of a curated set of add-ons for Amazon EKS clusters. 
+
+## Prerequisites
+
+Installing Operator as an add-on requires following prerequisites:
+
+* Ability to subscribe to [Datadog Operator][1] product. 
+* [TODO] License manager SLR.
+* Kubectl, AWS CLI if using command line interface.
+
+## Installing Operator
+
+There are certain differences when using add-on, compared to the regular [Helm installation][4]:
+* During Operator installation images must be pulled only from EKS repository, this can't be change by user.
+* Operator Helm Chart values which can be overriden is restricted by following [schema file][3].
+
+These restriction are necessary to make Operator compliant with the add-on policies, allow EKS ensure safety of the installation, and disable features not yet supported in add-on environment.
```

Comment:
i think this information should be in the top section. we should set the context up-front that there are two ways to install the Operator on EKS—helm chart and add-on—and that this document pertains to the add-on method. we should discuss what reasons a user may have to choose helm vs. add-on.

Reply:
Thanks for the feedback! Updated in https://github.com/DataDog/documentation/pull/19482/commits/e9180c6ca31e23458b28c5ee9dea2c5542a07d06, basically moved it the section to the top. Let me know what you think.